wrangler-service/src/main/java/co/cask/wrangler/service/schema/SchemaRegistryHandler.java

```diff
@@ -148,19 +132,24 @@ public void upload(HttpServiceRequest request, HttpServiceResponder responder,
   public void upload(HttpServiceRequest request, HttpServiceResponder responder, @PathParam("context") String namespace,
                      @PathParam("id") String id) {
     respond(request, responder, namespace, () -> {
-      byte[] bytes = null;
+      byte[] bytes;
       ByteBuffer content = request.getContent();
       if (content != null && content.hasRemaining()) {
         bytes = new byte[content.remaining()];
         content.get(bytes);
+      } else {
```

Comment:
why this change is needed?

Reply:
since bytes is now used in a closure, it needs to be effectively final